### PR TITLE
Add Micromax

### PR DESCRIPTION
**Pull request by Lost-Entrepreneur439**

### CODE REVIEW

File: brands/micromax/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Micromax
+
+* Verdict **ℹ️ "Safe for now" :trollface:**
+
+Despite their sketchy past with using their OTA updater app to [deliver adware][Micromax adware], Micromax follows the standard procedure of enabling OEM unlock in settings, then running `fastboot flashing unlock` in bootloader mode to unlock your bootloader.
```
Comment (melontini) [inline brands/micromax/README.md:5]: The boilerplate "standard unlock blah blah where you blah blah" should be replaced by a link to the generic unlock guide. `[**🔓️ Generic Unlock Guide**](/misc/generic-unlock.md)`

Comment (melontini) [inline brands/micromax/README.md:5]: The boilerplate "standard unlock blah blah where you blah blah" should be replaced by a link to the generic unlock guide. `/misc/generic-unlock.md`

File: brands/micromax/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Micromax
+
+* Verdict **ℹ️ "Safe for now" :trollface:**
```
Comment (melontini) [inline brands/micromax/README.md:3]: Should link the standard unlock guide.

```suggestion
* Verdict **ℹ️ "Safe for now" :trollface:**
* [**🔓️ Unlock Guide**](/misc/generic-unlock.md)
```

**Review by melontini — CHANGES_REQUESTED** [review]

File: brands/micromax/README.md (hunk 1 above)
Comment (Lost-Entrepreneur439) [reply brands/micromax/README.md]: fixed

**Review by melontini — APPROVED** [review]

**Pull request merged**